WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/FeeLineDetailsViewModel.swift

```diff
@@ -26,10 +20,29 @@ class FeeLineDetailsViewModel: ObservableObject {
         }
     }
 
-    /// Defines the amount text color.
+    /// Stores the percentage entered by the merchant.
     ///
-    var amountTextColor: UIColor {
-        amount.isEmpty ? .textPlaceholder : .text
+    @Published var percentage: String = "" {
+        didSet {
+            guard percentage != oldValue else { return }
+            percentage = priceFieldFormatter.formatAmount(percentage)
```

Comment:
Do we need to use the `priceFieldFormatter` on the percentage field, or is it ok if it's formatted differently? I don't have a strong opinion about this, either way.

Reply:
Good point, price formatter is too complex for this case, added dedicated `sanitizePercentageAmount` helper in 8f1ad17. Based on priceFieldFormatter, but without any custom settings.